docs/ecosystem/ifttt/readme.md

```diff
@@ -11,71 +11,48 @@ meta:
 
-[If This Then That (IFTTT)](https://ifttt.com) is a very popular web-based service that allows users to create "applets", which can combine different web services into powerful automation rules.
+# Getting Started
 
-## Why use openHAB with IFTTT?
+## Requirements
+* [openHAB Cloud Connector](https://www.openhab.org/addons/integrations/openhabcloud/#openhab-cloud-connector) configured using [myopenHAB.org](https://myopenHAB.org)
+* [IFTTT](https://ifttt.com/join) account
 
-openHAB already integrates with [far more than 200 different smart home technologies](/addons). While there are some web services like Twitter available, this is not the primary integration focus of openHAB. Therefore IFTTT perfectly complements the features of openHAB: With its [hundreds of channels](https://ifttt.com/channels), there are limitless new creative options: Be notified through text-to-speech if you are receive a DM on twitter, be warned through your lights if your favorite stock price falls below a certain threshold or put your house in away-mode when you turn on the ignition of your car - you can get further inspiration by browsing through existing applets.
+## Configuration
 
-<div class="row da-thumbs">
-  <article class="span4 bloc">
-    <a href="https://ifttt.com/recipes/300800-update-my-presence-if-nest-says-i-m-back-home">
-      <section class="img-wrapper">
-        <img alt="" src="./images/ifttt1.png" style="-webkit-transform: scale(1);">
-      </section>
-    </a>
+**1. Connect openHAB with IFTTT**
 
-  <div class="da-animate da-slideFromTop" style="display: block;">
-    <span class="iconWrapper iconLink icon-search" style="font-style: italic; margin-top: 63px"></span>
-  </div></a>
+IFTTT is available to all our users through the myopenHAB service. Activating the integration is easy. Just log in to your IFTTT account and activate the [openHAB service](https://ifttt.com/openhab). You will be forwarded to the [myopenHAB.org website](http://www.myopenhab.org/) to authorize the IFTTT service connection. 
+You can delete the IFTTT authorization token in the myopenHAB Applications section at any time.
 
-  <h3>Update my presence if Nest says I'm back home.</h3>
-  </article>
+**2. Expose specific items to IFTTT**
 
-  <article class="span4 bloc">
-    <a href="https://ifttt.com/recipes/297847-open-garage-when-i-turn-on-ignition">
-      <section class="img-wrapper">
-        <img alt="" src="./images/ifttt2.png" style="-webkit-transform: scale(1);">
-      </section>
-    </a>
+Before you start creating IFTTT applets you need to make sure that you configured your local openHAB runtime to expose certain items to myopenHAB.org. Only those items will be visible to IFTTT. Read here [how to expose items to myopenHAB.org](https://www.openhab.org/addons/integrations/openhabcloud/#configuration). You will also be able to send commands to those items from IFTTT Applets. 
```

Comment:
Small tidbit. Could you please have each sentence on its own line? The proper formatting is taken care of by the website builder.